pkgs/top-level/cuda-packages.nix

```diff
@@ -81,6 +81,61 @@ let
     nccl = final.callPackage ../development/cuda-modules/nccl { };
     nccl-tests = final.callPackage ../development/cuda-modules/nccl-tests { };
 
+    tests = {
```

Comment:
Is this `lib.cartesianProduct { openCVFirst = [ true false ]; useOpenCVDefaultCuda = [ true false ]; useTorchDefaultCuda = [ true false ]; }`?

Reply:
It is! Refactored in the latest.